contracts/scripts/deploy-registry.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * @notice Umbra Registry Deployment script
+ * @dev At each contract deployment or transaction, we wait for the transaction to be mined before
+ * continuing to the next step. This is done because it makes it simpler to continue from that spot
+ * rather than restart the full deployment
+ * @dev To initialize a deploy:
+ *   `yarn deploy:registry --network <network>`   (where network specifies a netwrk found in the hardhat.config.ts file)
+ * If deploying to a local node (--network localhost), first start Hardhat using `yarn hardhat node`
+ */
+const fs = require('fs');
+const hre = require('hardhat');
+const { exit } = require('process');
+const { ethers } = hre;
+
+const network = process.env.HARDHAT_NETWORK;
```

Comment:
Is this identical to `const network = hre.network.name`? I think so, but not as familiar with this approach so just confirming